frontend/src/theme/index.js

```diff
@@ -63,6 +63,13 @@ const theme = {
         color: '#2b8cc4',
       },
     },
+
+    footerLink: {
+      color: '#FFF',
```

Comment:
Can we use something like `theme.colors.white` here?

Reply:
good call! i'll open an issue to refactor WordMark & GoCSig to accept styled-system props